__tests__/app.test.js

```diff
@@ -190,6 +190,63 @@ describe("GET: /api/reviews", () => {
         });
       });
   });
+  test("200: returns with all reviews sorted by votes", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toHaveLength(13);
+        expect(body.reviews[0].votes).toBe(100);
+        expect(body.reviews[12].votes).toBe(1);
+        expect(body.reviews).toBeSortedBy("created_at", {
+          descending: true,
+          coerce: true,
+        });
+      });
+  });
+  test("200: returns with all the reviews ordered by ASC", () => {
+    return request(app)
+      .get("/api/reviews?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toHaveLength(13);
+        expect(body.reviews).toBeSortedBy("created_at");
+      });
+  });
+  test("200: returns with all the reviews with same category", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        body.reviews.forEach((review) => {
+          expect(review.category).toBe("dexterity");
+        });
```

Comment:
remember to include an assertion that the array (`reviews`) is not empty!

If it was empty your `forEach` would not invoke it's function and your jest test would automatically pass! 😱